pkg/es/config/config.go

```diff
@@ -271,14 +274,21 @@ func (c *Configuration) getConfigOptions() ([]elastic.ClientOptionFunc, error) {
 			}
 			httpTransport.TLSClientConfig = &tls.Config{RootCAs: ca}
 		}
+
+		token := ""
 		if c.TokenFilePath != "" {
-			token, err := loadToken(c.TokenFilePath)
+			tokenFromFile, err := loadToken(c.TokenFilePath)
```

Comment:
Does it mean that `TokenFilePath` and `AllowTokenFromContext` can't both be set at the same time? If that's the case, a WARN message should be printed out.